tests/Feature/Users/ViewUserTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Tests\Feature\Users;
+
+use Illuminate\Support\Facades\Notification;
+use App\Models\Company;
+use App\Models\User;
+use Tests\TestCase;
+use App\Notifications\CurrentInventory;
+
+class ViewUserTest extends TestCase
```

Comment:
All of these tests are named `UserWithoutCompanyPermissionsCannotX` but really they're all testing that users with the permission can get through while others are denied. We should update the test names to reflect that so many it's like `testPermissionsForX`.